src/Rest/File.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * This file is part of the PHP Rebilly API package.
+ *
+ * (c) 2015 Rebilly SRL
+ *
+ * For the full copyright and license information, please view the LICENSE.md
+ * file that was distributed with this source code.
+ */
+
+namespace Rebilly\Rest;
+
+/**
+ * Class File.
+ *
+ * @author Veaceslav Medvedev <veaceslav.medvedev@rebilly.com>
+ * @version 0.1
+ */
+final class File
+{
+    /** @var string */
+    private $content;
+
+    /** @var int */
+    private $size;
+
+    /** @var string */
+    private $mimeType;
+
+    /**
+     * @param string $mimeType
+     * @param string $content
+     * @param int|null $size
+     */
+    public function __construct($mimeType, $content, $size = null)
+    {
+        $this->mimeType = $mimeType;
+        $this->content = $content;
+        $this->size = $size ?: strlen($content);
```

Comment:
I know code works, but shouldn't we go with what we do right now using `isset` ?